js/main.js

```diff
@@ -17,19 +17,17 @@ function logForm(event) {
   var formObject = {
     title: $inputTitle.value,
     photoUrl: $imgUrl.value,
-    notes: $textArea.value,
-    nextEntryId: data.nextEntryId + 1,
-    view: 'entry-form',
+    notes: $textArea.value
   };
   data.nextEntryId += 1;
```

Comment:
You're incrementing `nextEntryId` but you don't have an entry id on the new entry anymore. You should make an `entryId` property and use the current value of `data.nextEntryId` as its value before you increment it.